dist/deploy/13_redemptionReceiver.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
@duncan4123 you don't need this contract right?

Reply:
correct

Reply:
don't add anything that won't be used